packages/roosterjs-content-model-plugins/lib/tableEdit/editors/features/TableRowColumnSelector.ts

```diff
@@ -0,0 +1,306 @@
+import { createElement } from '../../../pluginUtils/CreateElement/createElement';
+import { DragAndDropHelper } from '../../../pluginUtils/DragAndDrop/DragAndDropHelper';
+import { normalizeRect, parseTableCells } from 'roosterjs-content-model-dom';
+import type { CreateElementData } from '../../../pluginUtils/CreateElement/CreateElementData';
+import type { DragAndDropHandler } from '../../../pluginUtils/DragAndDrop/DragAndDropHandler';
+import type { Disposable } from '../../../pluginUtils/Disposable';
+import type { TableEditFeature } from './TableEditFeature';
+import type { OnTableEditorCreatedCallback } from '../../OnTableEditorCreatedCallback';
+import type {
+    DOMSelection,
+    IEditor,
+    ParsedTable,
+    ReadonlyContentModelTable,
+    Rect,
+    TableSelection,
+} from 'roosterjs-content-model-types';
+
+/**
+ * @internal
+ */
+export const ROW_SELECTOR_ID = 'rowSelector';
+/**
+ * @internal
+ */
+export const COLUMN_SELECTOR_ID = 'columnSelector';
+
+const STABLE_DOWN_ARROW_CURSOR =
+    'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgaGVpZ2h0PSIxNiIgdmlld0JveD0iMCAwIDE2IDE2Ij48dGV4dCB4PSI4IiB5PSIxMiIgdGV4dC1hbmNob3I9Im1pZGRsZSIgZm9udC1zaXplPSIxNCIgZmlsbD0iYmxhY2siPiYjMTI5MDk1OzwvdGV4dD48L3N2Zz4=';
+
+const STABLE_RIGHT_ARROW_CURSOR =
+    'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgaGVpZ2h0PSIxNiIgdmlld0JveD0iMCAwIDE2IDE2Ij48dGV4dCB4PSI4IiB5PSIxMiIgdGV4dC1hbmNob3I9Im1pZGRsZSIgZm9udC1zaXplPSIxNCIgZmlsbD0iYmxhY2siIHRyYW5zZm9ybT0icm90YXRlKC05MCA4IDgpIj4mIzEyOTA5NTs8L3RleHQ+PC9zdmc+';
+
+/**
+ * @internal
+ */
+export function createTableRowColumnSelector(
+    editor: IEditor,
+    table: HTMLTableElement,
+    isRowSelector: boolean,
+    anchorContainer?: HTMLElement,
+    onTableEditorCreated?: OnTableEditorCreatedCallback
+): TableEditFeature | null {
+    const rect = normalizeRect(table.getBoundingClientRect());
+    if (rect) {
+        const zoomScale = editor.getDOMHelper().calculateZoomScale();
+        const createElementData = getInsertElementData(rect, isRowSelector);
+        const div = createElement(createElementData, document) as HTMLDivElement;
+        div.id = isRowSelector ? ROW_SELECTOR_ID : COLUMN_SELECTOR_ID;
```

Comment:
Be careful, it is allowed to have multiple instances of editor on the same page. So do not use const id, but combine with editor id. We have a utility function to help do that. `ensureUniqueId` You may want to create a util in DOMHelper to help do this.

Reply:
This is used for unit test only, it is not needed to make the feature work.

Reply:
In that case, you can use a separate function to create this DIV, then mock it in test case so you can get that DIV in test code.

Adding ID here causes it appears in dev code and in production, others may write code to rely on it. And that may not work if there are multiple editor instances in the same page.